src/Symfony/Component/DependencyInjection/Extension/Extension.php

```diff
@@ -82,13 +81,8 @@ public function getConfiguration(array $config, ContainerBuilder $container)
         $namespace = $reflected->getNamespaceName();
 
         $class = $namespace.'\\Configuration';
-        if (class_exists($class)) {
-            $r = new \ReflectionClass($class);
-            $container->addResource(new FileResource($r->getFileName()));
```

Comment:
It hurts SF by default :) in terms of registering a duplicate resource. Imo. it just dont belong here..